lxd/main_init.go

```diff
@@ -494,8 +523,46 @@ they otherwise would.
 			return err
 		}
 	}
+	return nil
+}
+
+// Run the logic for preseed mode
+func (cmd *CmdInit) runPreseed(client lxd.ContainerServer) error {
+	data := &cmdInitData{}
+
+	if err := cmd.Context.InputYAML(data); err != nil {
```

Comment:
We don't use the `if err := foo(); err != nil {` syntax anymore and try to eliminate it wherever we see it. Please use:

```
err := foo()
if err != nil {
        return something
}
```

:)